### DISCUSSION TITLE
When is 2.2.1 coming to support react-native 65.1?

### DISCUSSION THREAD

**Discussion by 0nlyDev** [software-mansion/react-native-reanimated#2360] [Q&A] [answered]
Hi,

Would really like to know, before i decide to downgrade everything... 

Thank you!

**Comment by piaskowyk** [comment, marked as answer]
Hey, We have just released 2.3.0-alpha.3, so I'm working on the release of 2.2.1 today. It should be today or on monday.

**Reply by 0nlyDev** [reply]
Aweeesomeeee! Thank you!

**Reply by piaskowyk** [reply]
The package is ready. We will publish it next week.

**Reply by 0nlyDev** [reply]
@piaskowyk Can't wait! :)

**Reply by KoreanThinker** [reply]
@piaskowyk Great~!!!!! Thank you for manage this awesome library!!

**Reply by 0nlyDev** [reply]
> The package is ready. We will publish it next week.

> Hey, We have just released 2.3.0-alpha.3, so I'm working on the release of 2.2.1 today. It should be today or on monday.

Will the release be further delayed?

Thanks

**Reply by ralph-dev** [reply]
Also looking forward to this release, hopefully it comes with the missing Docs :) Would love to upgrade from V1!

**Comment by andreialecu** [comment]
`2.3.0-alpha.3` is still crashing on Android unfortunately: https://github.com/software-mansion/react-native-reanimated/issues/2256#issuecomment-912371769

Also looking forward to 2.2.1

**Comment by andreialecu** [comment]
So I went ahead and built this based on @piaskowyk's branch at https://github.com/software-mansion/react-native-reanimated/tree/%40piaskowyk/2.2.1 because we couldn't wait any longer 😄 .

Attaching it below in case it helps someone.

[react-native-reanimated-2.2.1.tgz.zip](https://github.com/software-mansion/react-native-reanimated/files/7138489/react-native-reanimated-2.2.1.tgz.zip)

It can be used by changing the entry in `package.json` to:
```json
    "react-native-reanimated": "file:./react-native-reanimated-2.2.1.tgz",
```

I can confirm that android builds no longer seem to be crashing.

You can also build it yourself by cloning the reanimated repo, switching to the 2.2.1 branch mentioned above, and running `./createNPMPackage.sh` (make sure you have gradle first, otherwise you may need to run `brew install gradle`)

PS: I assume the official release will come soon, so this should be very short lived and only meant for testing.

**Reply by 0nlyDev** [reply]
Thanks!
edit:

> It can be used by changing the entry in package.json to:
> 
>     "react-native-reanimated": "file:./react-native-reanimated-2.2.1.tgz",

What do you mean by that @andreialecu ? I can't figure it out..

**Reply by andreialecu** [reply]
It means to unzip the file I attached so you end up with the tgz file, put it somewhere and then update the entry in package json to point to it.

Run yarn install afterwards, and then rebuild via react-native run-android or however you have it set up.

**Comment by phalera** [comment]
Can it release today? thank you very very much!!!! @piaskowyk

**Comment by piaskowyk** [comment]
Yesterday I fixed some bugs from 2.2.1 - the package is ready to release.

**Comment by piaskowyk** [comment]
Here we go: https://github.com/software-mansion/react-native-reanimated/releases/tag/2.2.1 🚀

**Reply by numandev1** [reply]
@piaskowyk When is 2.3.0 coming?

**Reply by piaskowyk** [reply]
I'm not sure at this moment. We work on this but we still have a lot to do 💪
